### DISCUSSION TITLE
Updating 3rd party dependencies

### DISCUSSION THREAD

**Discussion by bjongbloedt** [danielgtaylor/huma#538] [General]
I was wondering if there was some automated system in place to update 3rd party dependencies?  I saw a few items that were out of date, and noticed dependabot ran at some point in the lifespan of the project with April-ish being the last run.

LMK if there is anything I can do to help, really enjoyed using the project so far.

**Comment by wolveix** [comment]
Hey @bjongbloedt, we fairly regularly update dependencies ourselves while working on other tasks. Sometimes dependabot will flag something urgent, though.
